environment.yml

```diff
@@ -21,6 +21,7 @@ dependencies:
   - click=8.1.7
   - tabulate=0.9.0
   - requests=2.31.0
+  - pytest=8.3.4
   - pip=24.3
   - pip:
```

Comment:
The below part can be deleted now as we've removed the conda-lock env set up option? As for docker option, this is all reflected in Dockerfile already.

```
- pip:
    - altair_ally==0.1.1
```

Reply:
removed both altair_ally and deepchecks